source/beamline/AMTemperatureMonitors.h

```diff
@@ -0,0 +1,43 @@
+#ifndef AMTEMPERATUREMONITORS_H
+#define AMTEMPERATUREMONITORS_H
+
+#include "beamline/AMConnectedControl.h"
+#include "beamline/AMTemperatureMonitorsStatus.h"
+
+class AMTemperatureMonitors : public AMConnectedControl
```

Comment:
I agree that the names are close enough to be confusing. Because the temperature monitors share so much in common with other controls like ion pumps, flow switches, pressure monitors, and flow transducers I actually don't expect this class to stick around for very long. I expect in #2127 this class will be entirely replaced with something more general, like `AMUtilitiesControlGroup`... or something. I'm having trouble coming up with a good name for that though. Something that's general enough to apply to all of those groups of controls, but not confusing.